pkg/scheduler/cache/util.go

```diff
@@ -18,9 +18,26 @@ package cache
 
 import (
 	"k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"strings"
 )
 
 // responsibleForPod returns true if the pod has asked to be scheduled by the given scheduler.
 func responsibleForPod(pod *v1.Pod, schedulerName string) bool {
 	return schedulerName == pod.Spec.SchedulerName
 }
+
+// convertNodeSelector converts node selector from string to *LabelSelector
+func convertNodeSelector(selector string) metav1.LabelSelector {
+	matchLabels := make(map[string]string)
+	if selector != "" {
+		s := strings.Split(selector, ":")
```

Comment:
IC, just support one label. Seems not so flexible.

Reply:
Do you have some idea about how to support multiple labels?

Reply:
1. If add `nodeSelector` as a scheduler parameter, maybe the form "<key1>:<value1>,<key2>:<value2>,..." works.
2. If in queue level, maybe just like how pod selects nodes, add a `nodeSelector` map to QueueSpec.